src/meta-srv/src/procedure/create_table.rs

```diff
@@ -197,15 +197,15 @@ impl CreateTableProcedure {
                 .map(|region_number| {
                     let region_id = RegionId::new(self.table_id(), *region_number);
 
-                    let mut create_table_request = request_template.clone();
-                    create_table_request.region_id = region_id.as_u64();
-                    create_table_request.region_dir = region_dir(catalog, schema, region_id);
+                    let mut create_region_request = request_template.clone();
+                    create_region_request.region_id = region_id.as_u64();
+                    create_region_request.region_dir = region_dir(catalog, schema, region_id);
 
-                    PbRegionRequest::Create(create_table_request)
+                    PbRegionRequest::Create(create_region_request)
                 })
                 .collect::<Vec<_>>();
 
-            create_table_tasks.push(common_runtime::spawn_bg(async move {
+            create_region_tasks.push(common_runtime::spawn_bg(async move {
```

Comment:
Why spawn here? Looks like the task won't be cancelled

Reply:
The spawned tasks are joined below, in line 221.

Reply:
I see. But you can also join without spawning it